src/pages/home/home.html

```diff
@@ -12,24 +12,10 @@
     <img src="assets/img/stockpile_tipi.svg" alt="Stockpile logo">
   </div>
 
-  <ion-segment [(ngModel)]="segment">
-    <ion-segment-button [value]="actions.rent">
-      {{actions.rent}}
-    </ion-segment-button>
-    <ion-segment-button [value]="actions.return">
-      {{actions.return}}
-    </ion-segment-button>
-  </ion-segment>
-
-	<ion-item>
-		<ion-input [(ngModel)]="tag" name="tag" type="text" placeholder="Tag">
-		</ion-input>
-	</ion-item>
-
-	<button ion-button class="tag-button" (click)="onNext()" type="submit" block [disabled]="!tag">
-    {{segment}}
+  <button ion-button (click)="onType()" block [outline]="platform.is('cordova')">
```

Comment:
I see. I would change it to be more descriptive, because `onType()` sounds like a handler that is activated when the user inputs text, but this handler is actually for clicking a button.

Since the button is "Type Barcode," a better function name might be `onTypeBarcode()`. Still not crystal clear, but at least consistent with the button text displayed in the UI.